demos/stock-securitydetails-asset-allocations/demo.js

```diff
@@ -0,0 +1,73 @@
+async function displayAssetAllocations (postmanJSON) {
+    const securityId = 'US4642898674';
+
+    const connector = new Connectors.Morningstar.SecurityDetailsConnector({
+        postman: {
+            environmentJSON: postmanJSON
+        },
+        security: {
+            id: securityId,
+            idType: 'ISIN'
+        },
+        type: 'AssetAllocations'
```

Comment:
How about setting this option on the `converter` level? What do you think @bre1470 ?